src/toast/tests/io_compression.py

```diff
@@ -0,0 +1,395 @@
+# Copyright (c) 2015-2021 by the parties listed in the AUTHORS file.
```

Comment:
Outdated years

Reply:
Fixed, but we should also just recursively update the ending copyright years across the codebase (as a separate, standalone PR).